identity-and-access-management/audit-trail/concepts.mdx

```diff
@@ -0,0 +1,45 @@
+---
+meta:
+  title: Audit Trail - Concepts
+  description: Learn fundamental concepts of Scaleway's Audit Trail.
+content:
+  h1: Concepts
+  paragraph: Learn fundamental concepts of Scaleway's Audit Trail.
+tags: audit-trail log events access
+dates:
+  validation: 2024-11-26
+---
+
+## Audit Trail
+
+Audit Trail is a tool that holds a record of events and changes performed within a Scaleway Organization. Audit Trail allows you to keep track of all actions, whether successful, attempted, or failed, perfomed on your Scaleway resources.
+
+With Audit Trail you can view actions taken by any [principal](/identity-and-access-management/iam/concepts/#principal) in any Project and on any of your Scaleway resources. You can also see the API methods used, the outcome of the action performed (status), and the IP address from which a principal performed an action.
+
+Audit Trail allows you to view the metadata of the API method used, allowing you to investigate incidents and troubleshoot issues.
+
+## Event
+
+An event is a record of any activity related to your Scaleway resources, encompassing actions by users, applications, or Scaleway itself.
+
+## Location
+
+In the context of Audit Trail, location refers to the place where the Scaleway resources being monitored are located. They can be located in different Scaleway Projects, or Organizations.
+
+## Method
+
+A method in the context of Audit Trail is the API call sent to the Audit Trail API whenever a principal takes an action (whether successful, attempted, or failed) on any Scaleway resource.
+
+## Region
+
+A region is the geographical area in which your resources are located (Paris, Amsterdam, or Warsaw).
+
+Audit Trail allows you to select and filter your resources by region and Scaleway Project. This allows you to better sort through your resources.
+
+## Status
+
+A status is the API response returned whenever an action was taken on a resource by a principal. There are two statuses:
+
+- `200`: this means that the API request was executed.
+
+- `500`: this means that the API request has failed.
```

Comment:
We can add the `403` status for permission denied

Reply:
I'm not sure we want to document 500 because these are for internal server error, where the error occurs on the Scaleway servers due to a bug or an incident.